src/app-layer-events.c

```diff
@@ -135,6 +143,7 @@ void AppLayerDecoderEventsSetEvent(Flow *f, uint8_t event)
     AppLayerDecoderEvents *new = events;
     AppLayerDecoderEventsSetEventRaw(&events, event);
     if (events != new)
+        SCLogNotice("Setting events to %p", events);
```

Comment:
Probably shoudln't be notice.

Reply:
You're correct -- I'll remove this line. I don't think it's useful for debugging either.